src/modules/quizzes.js

```diff
@@ -35,12 +35,12 @@ function createQuizUrl(quizId, parameters, options, path) {
     throw new Error('quizId is a required parameter of type string');
   }
 
-  if (path === 'finalize' && (typeof parameters.answers !== 'object' || !Array.isArray(parameters.answers) || parameters.answers.length === 0)) {
+  if (path === 'results' && (typeof parameters.answers !== 'object' || !Array.isArray(parameters.answers) || parameters.answers.length === 0)) {
     throw new Error('answers is a required parameter of type array');
   }
 
   if (parameters) {
-    const { section, answers, versionId } = parameters;
+    const { section, answers, versionId, page, resultsPerPage, filters } = parameters;
```

Comment:
reminder to destruct quiz_session_id here as well?

Reply:
Handled here https://github.com/Constructor-io/constructorio-client-javascript/pull/245/